spock-specs/src/test/groovy/org/spockframework/docs/extension/TempDirDocSpec.groovy

```diff
@@ -16,7 +16,7 @@ class TempDirDocSpec extends Specification {
     expect: true
```

Comment:
The only thing left is that this spec should at least prove that the fields were injected, e.g.

```groovy
def demo() {
  expect:
  path1 instanceof Path
  path2 instanceof File
  path3 instanceof Path
}
```

if you move the method below the fields you can add it to the examples to illustrate what is injected